docsrc/source/api/api_docs/modules/hardware_representation.rst

```diff
@@ -0,0 +1,340 @@
+:orphan:
+
+.. _ug-hardware_representation:
+
+
+=================================
+hardware_representation Module
+=================================
```

Comment:
Please add a note the current we only support Quantizer type in HM

Reply:
Fixed